_includes/about.html

```diff
@@ -29,6 +29,8 @@
           commissioned to write/record the theme song for author Tony Bowman’s audiobook
           version of his award-winning novel, Valkyrie: The Road. We were also commissioned
           to write/record a tribute song for the Southern Wake Relay For Life in 2016.
+
```

Comment:
Do you want to put some br tags here to make it a new line?

Reply:
Yes.  Just testing.